gdx/src/com/badlogic/gdx/ApplicationListener.java

```diff
@@ -16,6 +16,8 @@
 
 package com.badlogic.gdx;
 
+import com.badlogic.gdx.utils.Disposable;
```

Comment:
This can be removed again.

I'm starting to think the `Disposable` `Screen` change makes sense. Doesn't really matter that it overrides the `Disposable` interface's `dispose()` - `Music` and probably other interfaces do that as well. Nice pull request.